app/templates/auth/change-password.html

```diff
@@ -0,0 +1,122 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Change password – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": dashboard_url,
+        "label": "Your account"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+{% if form.errors|length > 1 %}
+    <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+        <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+            There was a problem with the details you gave for:
+        </h3>
+        <ul>
+        {% for field_name, field_errors in form.errors|dictsort|reverse if field_errors %}
+          {% for error in field_errors %}
+            <li><a href="#{{ form[field_name].name }}" class="validation-masthead-link">{{ form[field_name].label.text }}</a></li>
+          {% endfor %}
+        {% endfor %}
+        </ul>
+    </div>
+{% endif %}
+
+<header class="page-heading">
+    <h1>Change your password</h1>
+</header>
+
+<form autocomplete="off" action="{{ url_for('.change_password') }}" method="POST">
+
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            {{ form.hidden_tag() }}
+
+
+            {% if form.old_password.errors %}
+                <div class="validation-wrapper">
+            {% endif %}
+                <div class="question" id="{{ form.old_password.name }}">
+                    {{ form.old_password.label(class="question-heading") }}
+                    {% if form.old_password.errors %}
+                    <p class="validation-message" id="error-old-password-textbox">
+                        {% for error in form.old_password.errors %}{{ error }}{% endfor %}
+                    </p>
+                    {% endif %}
+                    {{ form.old_password(class="text-box", autocomplete="off") }}
+                </div>
+            {% if form.old_password.errors %}
+                </div>
+            {% endif %}
+
+
+            {% if form.password.errors %}
```

Comment:
This template only gets passed a `ChangeOldPasswordForm` which only has form.old_password I think.

Reply:
Is this template replacing the `reset-passord.html` template? If so, does `reset_password.py` need updating?